### PR TITLE
[Yaml] Added support for multiple files or directories in LintCommand

### PR DESCRIPTION
**Pull request by yceruto**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #28498
| License       | MIT
| Doc PR        | https://github.com/symfony/symfony-docs/pull/10367

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/composer.json [hunk 1]
```diff
@@ -74,7 +74,8 @@
         "symfony/translation": "<4.2",
         "symfony/twig-bridge": "<4.1.1",
         "symfony/validator": "<4.1",
-        "symfony/workflow": "<4.1"
+        "symfony/workflow": "<4.1",
+        "symfony/yaml": "<=4.2"
```
Comment (xabbuh) [inline src/Symfony/Bundle/FrameworkBundle/composer.json]: `<4.2`

**Comment by nicolas-grekas** [comment]
deps=high failures mean this PR break 4.1 when used with deps at 4.2 - this should be fixed somehow (yes, that's the big downside of moving things to components: we now have to deal with cross deps compat...)

**Comment by stof** [comment]
Well, here, the issue is only related to the way our CommandTester works. Turning an argument into an array argument is a BC break for tests, because the argument now has to be provided as an array, while there is no BC break when running the command in the CLI.

And as FrameworkBundle extends the command and has tests for its child class, it is affected by this change in the argument definition of the component.

**Comment by yceruto** [comment]
Could be an alternative to do this in the command `$filenames = (array) $input->getArgument('filename');` and revert all changes in tests?

File: src/Symfony/Bundle/FrameworkBundle/composer.json (hunk 1 above)
Comment (yceruto) [reply src/Symfony/Bundle/FrameworkBundle/composer.json]: Done, but still:
> - Installing symfony/yaml (4.2.x-dev): Downloading (100%)

Comment (nicolas-grekas) [reply src/Symfony/Bundle/FrameworkBundle/composer.json]: which mean it worked actually

**Review by chalasr — APPROVED** [review]
got it :)

File: src/Symfony/Bundle/FrameworkBundle/composer.json (hunk 1 above)
Comment (yceruto) [reply src/Symfony/Bundle/FrameworkBundle/composer.json]: This is the fails before the last commit https://travis-ci.org/symfony/symfony/jobs/430972569#L2994

**Review by nicolas-grekas — APPROVED** [review]

**Review by javiereguiluz — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @yceruto.

**Pull request merged**